scripts/pre_commit_linter.py

```diff
@@ -1540,30 +1540,30 @@ def _check_sorted_dependencies(all_files, parsed_js_files):
                             'Please ensure that %s in %s, the injected '
                             'dependecies should be in the following manner:'
                             ' dollar imports, regular imports and constant '
-                            'imports, all in sorted form.'
+                            'imports, all in sorted order.'
                             % (property_value, filename))
                     if sorted_imports != literal_args:
                         failed = True
                         print (
-       	                    'Please ensure that %s in %s, the dependecies '
-       	                    'literally mentioned should be in the following '
-       	                    ' manner: dollar imports, regular imports and '
-       	                    'constant imports, all in sorted form.'
+       	                    'Please ensure that %s in %s, the stringfied '
```

Comment:
Read the previous comment _carefully_!

Reply:
Hi @apb7 is the error message correct, or needs to be modified ?